src/components/paradox/results/personality/career/inline-jobs/index.js

```diff
@@ -0,0 +1,138 @@
+import PropTypes from "prop-types";
+import {faCaretLeft, faCaretRight} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import {useEffect, useState} from "react";
+import InlineJob from "./job";
+import style from "./style.scss";
+
+function CarouselButton({className, onClick, children, disabled}) {
+  const [isDisabled, setIsDisabled] = useState(disabled);
```

Comment:
Any reason `isDisabled` is different than `disabled`? Looks like you can just use `disabled`

Reply:
good catch, removed!